src/dev/flang/be/jvm/classfile/ClassFileConstants.java

```diff
@@ -0,0 +1,888 @@
+/*
+
+This file is part of the Fuzion language implementation.
+
+The Fuzion language implementation is free software: you can redistribute it
+and/or modify it under the terms of the GNU General Public License as published
+by the Free Software Foundation, version 3 of the License.
+
+The Fuzion language implementation is distributed in the hope that it will be
+useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+License for more details.
+
+You should have received a copy of the GNU General Public License along with The
+Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+*/
+
+/*-----------------------------------------------------------------------
+ *
+ * Tokiwa Software GmbH, Germany
+ *
+ * Source of class ClassFileConstants
+ *
+ *---------------------------------------------------------------------*/
+
+package dev.flang.be.jvm.classfile;
+
+import dev.flang.util.ANY;
+
+
+/**
+ * ClassFileConstants provides constants used in a Java class file.
+ *
+ + Please refer to the Java Virtual Machine Specification for details,
+ * latest version as time of writing available here
+ *
+ *   https://docs.oracle.com/javase/specs/jvms/se20/jvms20.pdf
+ *
+ * @author Fridtjof Siebert (siebert@tokiwa.software)
+ */
+public interface ClassFileConstants
+{
+
+
+  /*----------------------------  constants  ----------------------------*/
+
+
+  public static final byte[] MAGIC = new byte[] { (byte) 0xCA, (byte) 0xFE, (byte) 0xBA, (byte) 0xBE };
+
+  public static byte[] VERSION_JDK_1_0 = new byte[] { 0, 0, 0, 45 };
+  public static byte[] VERSION_JDK_1_1 = new byte[] { 0, 3, 0, 45 };
```

Comment:
the 3 in `0, 3, 0, 35` seems odd?

Reply:
This is correct, in the early days of Java the minor version was used.